crates/dslab-mp/src/logger.rs

```diff
@@ -422,3 +260,171 @@ pub enum LogEntry {
         timer: String,
     },
 }
+
+impl LogEntry {
+    fn print(&self) {
```

Comment:
Надо сделать публичной, чтобы можно было вызвать из тестов

```suggestion
    pub fn print(&self) {
```